### PR TITLE
fix: Update stored message to give out only the latest message

### PR DESCRIPTION
**Pull request by edwinjosechittilappilly**
Update stored message to give out only the latest message.
This pull request includes changes to the `StoreMessageComponent` class in the `store_message.py` file to improve the handling of stored messages. The most important changes include renaming an output and modifying the `store_message` method to use more efficient message retrieval logic.

Changes to output naming:

* [`src/backend/base/langflow/components/helpers/store_message.py`](diffhunk://#diff-c873d4f3dd80c34e0163a9060b7a116f076e62e99e5105a0fe11b4e06f99a0faL47-R47): Renamed the output from "Stored Messages" to "Stored Message" to reflect that only a single message is stored.

Improvements to message retrieval logic:

* [`src/backend/base/langflow/components/helpers/store_message.py`](diffhunk://#diff-c873d4f3dd80c34e0163a9060b7a116f076e62e99e5105a0fe11b4e06f99a0faL65-R73): Modified the `store_message` method to use the `next` function for retrieving messages, ensuring that only the first matching message is returned, and added a default value of `None` if no messages match.

### CODE REVIEW

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]
Hey @edwinjosechittilappilly 

Why was this done? Can't we just get `store[0]`?

**Comment by edwinjosechittilappilly** [comment]
@ogabrielluiz  
I had corrected it with the suggested approach. Earlier, I used next to provide the default case. Yea, but you were right.

**Review by anovazzi1 — APPROVED** [review]
lgtm

**Comment by edwinjosechittilappilly** [comment]
@ogabrielluiz  can you review it once available?

File: src/backend/base/langflow/components/helpers/store_message.py [hunk 1]
```diff
@@ -70,5 +70,5 @@ async def store_message(self) -> Message:
             stored = await aget_messages(
                 session_id=message.session_id, sender_name=message.sender_name, sender=message.sender
             )
```
Comment (ogabrielluiz) [inline src/backend/base/langflow/components/helpers/store_message.py:72]:
```suggestion
            stored_messages = await aget_messages(
                session_id=message.session_id, sender_name=message.sender_name, sender=message.sender
            )
            if not stored_messages:
            	raise SomeError
            stored_message = stored_messages[0]
            
            
```

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]

**Comment by edwinjosechittilappilly** [comment]
@ogabrielluiz  I had made the changes requested.

**Review by ogabrielluiz — APPROVED** [review]

**Pull request merged**